infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -68,6 +65,12 @@ void prepare() throws InfrastructureException {
       createExecRole(k8sClient, execRoleName);
     }
 
+    String viewRoleName = "view";
```

Comment:
Are you going to change role name to `workspace-view` https://github.com/eclipse/che/pull/12789#discussion_r261084245 ?

Reply:
Yes, renamed.